rollingpin/main.py

```diff
@@ -153,9 +162,18 @@ def _main(reactor, *raw_args):
     # the cypher used for SSH connections. we don't care and can't do anything
     # about it for now.
     warnings.simplefilter("ignore")
+    initial_parser = make_profile_parser(PROFILE_DIRECTORY)
+    if not raw_args:
+        initial_parser.print_help()
+        sys.exit(0)
+    args, raw_args = initial_parser.parse_known_args(args=raw_args)
+
+    config = _load_configuration(args.profile, PROFILE_DIRECTORY)
+    args = _parse_args(config, raw_args, args.profile)
+    if args.test:
```

Comment:
I like it. If you really wanted to get fancy you could explain in english what it'll do:

```
root@rollingpin.lxc:# rollout example --test      
rollout -h none --parallel=2 --timeout=60 -d rollingpin -r all

This will:

* Deploy the `rollingpin` component
* Restart `all` applications on the target hosts.

to `none` hosts 2 at a time, timing out if any host takes longer than 60 seconds.
```